astropy/modeling/core.py

```diff
@@ -2877,9 +2877,13 @@ def _format_str(self, keywords=[], defaults={}):
 
         if columns:
             param_table = Table(columns, names=self.param_names)
-            # Set units on the columns
+
```

Comment:
I guess this comment was removed by mistake ? Can you add it back ?

Reply:
Sorry! I will add it back. Thanks for noticing.